content/news/2021-04-04-bevy-0.5/index.md

```diff
@@ -1119,6 +1119,19 @@ Naively, it would also _disallow_ `filter_system`, which would be a significant
 
 This means `filter_system` is still perfectly valid in **Bevy 0.5**. I consider this a "best of both worlds" situation. We get most of the benefits of the old implementation, but with consistent and predictable rules enforced at app startup. 
 
+## What's Next For Bevy?
+
+We still have a long road ahead of us, but the Bevy developer community is growing at a rapid pace and we already have big plans for the future. Expect to see progress in the following areas soon:
+
+* "Pipelined" rendering and other renderer optimizations
+* Bevy UI redesign
+* Animation: component animation and 3d skeletal animation
+* ECS: relationships/indexing, async systems
```

Comment:
archetype invariants!

Reply:
Also Ratys's stageless work

Reply:
I believe in no more `.system()`. Maybe let's not promise it though